patches/api/0469-API-for-checking-sent-chunks.patch

```diff
@@ -0,0 +1,42 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Flo0 <flo.roma@web.de>
+Date: Mon, 8 Apr 2024 16:22:07 +0200
+Subject: [PATCH] API for checking sent chunks
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index c6cb4f17469a8f2e60dd3e28d41402851ce5fb21..ecf4542971b0293b3105567a45689f434f04318e 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -3707,6 +3707,31 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+     void resetIdleDuration();
+     // Paper end
+ 
++    // Paper start - Add chunk view API
++    /**
++     * Gets the set of chunks that have been sent to the player.
++     *
++     * @return an immutable set of chunk keys
++     */
++    java.util.Set<java.lang.Long> getSentChunkKeys();
```

Comment:
Can be marked with @org.jetbrains.annotations.Unmodifiable.
Additionally, the NotNull annotation.

Reply:
Unmodifiable has a retention policy of class only i think

Reply:
Correct. It isn't needed at runtime, but Intellij can infer more about the returned collection via said annotation.